src/applications/personalization/profile-2/profile-2-entry.jsx

```diff
@@ -1,8 +1,9 @@
-import './sass/profile-2.scss';
 import 'platform/polyfills';
-
 import startApp from 'platform/startup';
 
+import './sass/profile-2.scss';
+
+import profileUi from './reducers';
```

Comment:
Can we combine this reducer with the one in `profile360`?

Reply:
I'm gonna take a fresh look at how I'm storing this data in Redux in a follow up PR where I also add Redux tests.